packages/ra-keycloak/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Many duplicate lines between this `packages/ra-keycloak/package.json` and the root `package.json`.

I believe the root one should only declare a name, the workspace, scripts, and common dependencies.

All info related to the ra-keycloak package (version, license, gihub and stuff) should only be in `packages/ra-keycloak/package.json` IMO.

You may want to have a look at how it's done in the react-admin repo.